src/main/target/common_pre.h

```diff
@@ -330,6 +330,10 @@
 #define USE_THRUST_LINEARIZATION
 #define USE_TPA_MODE
 
+#if defined(USE_SERIALRX_SRXL2) && !defined(USE_SERIALRX_SPEKTRUM)
+#define USE_SERIALRX_SPEKTRUM
+#endif
+
 #ifdef USE_SERIALRX_SPEKTRUM
```

Comment:
Did you try this:
```suggestion
#if defined(USE_SERIALRX_SPEKTRUM) || defined(USE_SERIALRX_SRXL2)
```

Reply:
Would work if we change it here too

https://github.com/betaflight/betaflight/blob/f89028759806d60223b8e946bf096b8d49a4b368/src/main/target/common_post.h#L268-L282